src/vendors/oasisscan.ts

```diff
@@ -108,7 +106,14 @@ export function getOasisscanAPIs(url: string | 'https://api.oasisscan.com/mainne
     if (!transactionsList || transactionsList.code !== 0) throw new Error('Wrong response code') // TODO
 
     const list = await Promise.all(
-      transactionsList.data.list.map(async tx => (tx.runtimeId ? getRuntimeTransactionInfo(tx) : tx)),
+      transactionsList.data.list.map(async tx => {
+        if (tx.runtimeId) {
+          return await getRuntimeTransactionInfo(tx)
+        } else {
+          const { nonce } = await getTransaction({ hash: tx.txHash })
+          return { ...tx, nonce }
+        }
+      }),
```

Comment:
One difference from before: this fails if any request fails, unlike allSettled. But I think simplifying it is worth it

Reply:
I kind of liked, that it did not fail before if any detailed tx call fails. As it would still work in most cases, but sure, this way there is no flakiness.